### PR TITLE
Ensure DB ID is set if namespaces are enabled

### LINKED ISSUE #765: Namespace creation accepts empty bottomless_db_id

### PR DESCRIPTION
**Pull request by avinassh**
closes #765

### CODE REVIEW

File: bottomless/src/replicator.rs [hunk 1]
```diff
@@ -155,12 +155,19 @@ impl Options {
...
             match std::env::var(key) {
-                Ok(res) => Ok(res),
+                Ok(res) => {
+                    let res = res.trim().to_string();
+                    if res.is_empty() {
+                        bail!("{} environment variable is empty", key)
+                    } else {
+                        Ok(res)
+                    }
+                }
```
Comment (avinassh) [inline bottomless/src/replicator.rs:165]: If we don't trim, then it lets us slip in values containing just spaces. I don't think we ever want env variables to have spaces, trimming seems like a must.

**Review by psarna — APPROVED** [review]

File: bottomless/src/replicator.rs [hunk 2]
```diff
@@ -155,12 +155,19 @@ impl Options {
     pub fn from_env() -> Result<Self> {
         fn env_var(key: &str) -> Result<String> {
```
Comment (psarna) [inline bottomless/src/replicator.rs:156]: I preemptively acked, but before we merge -- should all `env_var` callsites disallow empty vars?

Comment (psarna) [reply bottomless/src/replicator.rs:156]: @avinassh can you double-check before I queue this?

Comment (avinassh) [reply bottomless/src/replicator.rs:156]: @psarna hey, let me double check all of the vars if we are using empty vars anywhere

Comment (avinassh) [reply bottomless/src/replicator.rs:156]: @psarna I have double checked all, we don't expect empty variables to be set anywhere in bottomless

**Pull request merged**